### ISSUE TITLE
How to generate bvh from videos

### ISSUE THREAD

**Issue by shichaog** [GestureGeneration/Speech_driven_gesture_generation_with_autoencoder#9]
This is an excellent work. As a student working on gesture generation, I am very appreciate of this code. Thank you.

However I've got a question for you guys.

I have a collections of video files, how can I generate bvh from these mp4 files?

Looking forward to your reply!

**Comment by Svito-zar** [comment]
Hi @shichaog, that you for your interest in the work.
What you are asking is basically how to obtain 3D motion from 2D videos, which is also referred to as _3D pose estimation_.  It is a very challenging task, which has been investigated for several decades.
These are some of the most recent methods I know:
- https://penincillin.github.io/frank_mocap
- https://europe.naverlabs.com/blog/dope-distillation-of-part-experts-for-whole-body-3d-pose-estimation-in-the-wild/
- https://zju3dv.github.io/iMoCap/
- https://crockwell.github.io/partial_humans/

So in short - to obtain a BVH file from a video is highly not trivial, but possible (still with low quality) using recent tools.
Hope it helps.
